.github/workflows/scan-vulns.yaml

```diff
@@ -10,6 +10,8 @@ on:
       - "docs/**"
       - "library/**"
       - "**.md"
+  schedule:
+    - cron: '30 8 * * 0'  # early morning (08:30 UTC) every Sunday
```

Comment:
@akashsinghal , wondering if this schedule only runs on the dev branch?

Reply:
According to this [doc](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), scheduled workflows will only run on the default branch.

Reply:
I think we would likely need a separate workflow then for the release branch scanning. I'm wondering if there's a way to get the last release branch and check that out. Some sort of github variable that would allow that. In that case, we could then still have the default branch trigger but it will checkout the correct release version.

Reply:
If this is too much work, maybe we forgo the schedule and just merge the existing scan vulnerability workflow into the release branch so that at least we can run it manually when needed.

Reply:
curious which branch and what time we would like to run the scan against? Release branch before releasing?

Reply:
as discussed in CC: this will only run on dev branch but we still find it valuable to run once a week. especially when new PRs haven't merged to dev (e.g holidays)